packages/gated-content/tsconfig.base.json

```diff
@@ -1,24 +1,10 @@
 {
+  "extends": "@lens-protocol/tsconfig/base.json",
   "compilerOptions": {
-    "allowJs": true,
-    "declaration": true,
-    "declarationMap": true,
-    "esModuleInterop": true,
-    "forceConsistentCasingInFileNames": true,
-    "importHelpers": true,
     "lib": ["DOM", "ESNext"],
     "module": "ESNext",
-    "moduleResolution": "Node",
-    "noFallthroughCasesInSwitch": true,
-    "noImplicitReturns": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
-    "noUncheckedIndexedAccess": true,
-    "resolveJsonModule": true,
-    "skipLibCheck": true,
-    "sourceMap": true,
-    "strict": true,
-    "target": "esnext"
+    "outDir": "dist"
   },
-  "include": ["./src"]
+  "include": ["./src"],
+  "exclude": ["./node_modules"]
```

Comment:
do you need to exclude `./node_modules` if we are already not including them on `include` side?

Reply:
let me clean this, just comes from an iteration while I was experimenting custom build with rollup.